conans/client/build/cmake.py

```diff
@@ -337,6 +337,10 @@ def add_cmake_flag(cmake_flags, name, flag):
                 shared = self._conanfile.options.get_safe("shared")
                 ret["CONAN_CMAKE_POSITION_INDEPENDENT_CODE"] = "ON" if (fpic or shared) else "OFF"
 
+        # Adjust automatically the module path in case the conanfile is using the cmake_find_package
+        if "cmake_find_package" in self._conanfile.generators:
+            ret["CMAKE_MODULE_PATH"] = self._conanfile.install_folder
```

Comment:
Shouldn't be a list, so it is appended, and other CMAKE_MODULE_PATH are respected?